src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/dropdown.scss

```diff
@@ -0,0 +1,49 @@
+@import '../Application/colors.scss';
+
+$dropdownBackgroundColor: $wildSand;
+$itemTextActiveColor: $shakespeare;
+
+.container {
+    display: none;
+    position: absolute;
+    left: 0;
+    top: 100%;
+    z-index: 1;
+    padding-right: 20px;
+    padding-left: 52px;
```

Comment:
Until now I have tried to always us the `padding` shorthand. Maybe we should agree on that?